### ISSUE TITLE
ios-sim not found error - but it is installed

### ISSUE THREAD

**Issue by richkirsch** [driftyco/ionic-lab-issues#54]
Attempting to either run or simulate an app results in the following error:

<img width="420" alt="screen shot 2015-08-18 at 19 59 41" src="https://cloud.githubusercontent.com/assets/5949743/9340522/0e9175e4-45e8-11e5-84b9-397d2fcdc2fe.png">

This is the case both with new apps and existing apps. I can run ios-sim in the terminal (version 4.1.1). Everything works as expected using ionic CLI, such as ionic emulate ios.

I had previously installed ios-sim with 'npm install -g ios-sim'. However, possibly the issue is something to do with the fact that I struggled with permissions when first using npm so, following the official advice on <a href="https://docs.npmjs.com/getting-started/fixing-npm-permissions#option-2-change-npm-s-default-directory-to-another-directory">this page</a> I changed my npm's default directory to another directory (in which all my nom packages are installed including ionic, cordova and ios-sim).

**Comment by aaronallport** [comment]
+1

I'm having this exact issue too - any advice would be appreciated. Thanks!

**Comment by jbavari** [comment]
Would you please run `npm install -g ios-sim` and see if that solves things?

I'm wondering if the latest version fixes the issue.

**Comment by richkirsch** [comment]
No, exactly the same error. I'm feel like it must be ionic lab not knowing to access my (changed) npm global directory, but I don't know how to correct for this.

**Comment by nicon-dev** [comment]
+1
same error here. I did `npm install -g ios-sim` but it still shows the same error message.

`ios-sim --version
5.0.1`
When I run `ionic emulate ios` the ios emulator shows up and it works.

And one other thing: I have ios & android platform installed. When I click "emulate" should it start both emulators? Would be good to select ether ios/android to start/emulate.

```
ionic info

Your system information:

Cordova CLI: 5.3.1
Gulp version:  CLI version 3.9.0
Gulp local:   Local version 3.9.0
Ionic Version: 1.1.0
Ionic CLI Version: 1.6.4
Ionic App Lib Version: 0.3.8
ios-deploy version: 1.7.0 
ios-sim version: 5.0.1 
OS: Mac OS X Yosemite
Node Version: v0.12.7
Xcode version: Xcode 6.4 Build version 6E35b 
```

**Comment by brettcorey** [comment]
Same issue.

Cordova CLI: 5.3.1
Gulp version:  CLI version 3.9.0
Gulp local:  
Ionic Version: 1.1.0
Ionic CLI Version: 1.6.4
Ionic App Lib Version: 0.3.8
ios-deploy version: 1.7.0 
ios-sim version: 5.0.1 
OS: Mac OS X Yosemite
Node Version: v0.12.4
Xcode version: Xcode 7.0 Build version 7A220

**Comment by mussrya** [comment]
I have the same issue on mine, it seems like others have said it looks like that it's referencing a different directory. I modified the ionic application on mine as follows to fix the issue:
Right click on Ionic Lab.app and click show package contents
Contents > Resources > app > js > nodeservice.js

Changing this line:

```
  process.env.PATH = process.env.PATH + ':/usr/local/bin';
```

to:

```
  process.env.PATH = process.env.PATH + ':/usr/local/bin:/Users/username/.npm-packages/bin';
```

This resolved it for me as a temporary solution at least.

**Comment by divanvisagie** [comment]
I did something similar:

  //process.env.PATH = process.env.PATH + ':/usr/local/bin';
  process.env.PATH = process.env.PATH + ':/Users/username/.nvm/versions/node/v4.2.2/bin/node';

Which solved my #73 which was actually related to #50
